### PR TITLE
Nicholas/buttonfix

### PR DESCRIPTION
**Pull request by Nap1400**
There was an error on the button component that had the default size and height to size.m which doesn't exist. I changed it to size.rg, which does.

### CODE REVIEW

**Review by mshago — APPROVED** [review]

**Review by carizzuti — APPROVED** [review]
I expected this to happen with the new size file.

**Review by aahenry51 — APPROVED** [review]
Got it, you just changed the size.

File: mobile/src/components/categoryRadio/CategoryRadio.styles.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import {StyleSheet} from 'react-native';
+export const styles =
+StyleSheet.create({
+    container: {
+        backgroundColor: '',
+        borderRadius: 10,
+        paddingHorizontal: 10, 
+        paddingVertical: 10,
```
Comment (raynesmike) [inline mobile/src/components/categoryRadio/CategoryRadio.styles.ts:8]: since we fixed the size theme you can start adding them now

**Review by raynesmike — APPROVED** [review]
I approve this

**Pull request merged**